lib/model/autocomplete.dart

```diff
@@ -43,6 +44,16 @@ extension Autocomplete on ComposeContentController {
   }
 }
 
+extension ComposeTopicAutocomplete on ComposeTopicController {
+  AutocompleteIntent<TopicAutocompleteQuery>? autocompleteIntent() {
+    if (!selection.isValid || !selection.isNormalized) return null;
```

Comment:
Is there a reason we need these checks? It doesn't seem like we're using the selection in the remaining logic.